src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -200,6 +221,21 @@ export default {
 				});
 			}
 
+			// track cash 794 if shown
+			if (this.showFavoriteCountryRow) {
+				const favoriteCountryLoans = _get(this.$refs, 'favoriteCountries.favoriteCountryLoans');
+				if (favoriteCountryLoans !== undefined && favoriteCountryLoans.length > 0) {
+					_each(favoriteCountryLoans, (loan, loanIndex) => {
+						loanIds.push({
+							r: -1,
+							p: loanIndex + 1,
+							c: 99,
+							l: loan.id
+						});
+					});
```

Comment:
r:c:p:l:, this is nice. 👍

Reply:
This pushes data into our custom page state event alongside all the other rows. It is then compared with an interaction event containing the same information when an add to basket button is clicked. Thus we can build a map of which rows and positions are most affective.